torch/fx/subgraph_rewriter.py

```diff
@@ -60,25 +63,52 @@ def attributes_are_equal(pn : Node, gn : Node) -> bool:
             return False
         match_found = all(self._match_nodes(pn_, gn_) for pn_, gn_
                           in zip(pn.all_input_nodes, gn.all_input_nodes))
+        match_found = False
+        if pn.op == "output":
+            # This covers the case in which the original graph Node that
+            # matched to the pattern subgraph's output has more than
+            # one input Node
+            for perm in list(permutations(gn.all_input_nodes)):
+                match_found = all(self._match_nodes(pn_, gn_) for pn_, gn_
+                                  in zip(pn.all_input_nodes, perm))
+                if match_found:
+                    break
+        else:
+            match_found = (len(pn.all_input_nodes) == len(gn.all_input_nodes)
+                           and all(self._match_nodes(pn_, gn_) for pn_, gn_
+                                   in zip(pn.all_input_nodes, gn.all_input_nodes)))
         if not match_found:
             self.nodes_map.pop(pn)
             return False
 
         return True
 
 
-def replace_pattern(gm : GraphModule, pattern : Callable, replacement : Callable) -> None:
+def replace_pattern(gm : GraphModule, pattern : Callable, replacement : Callable) -> List[Match]:
     """
     Matches all possible non-overlapping sets of operators and their
     data dependencies (``pattern``) in the Graph of a GraphModule
     (``gm``), then replaces each of these matched subgraphs with another
     subgraph (``replacement``).
 
-    Args:
+    Parameters:
```

Comment:
Shouldn't this be `Args` under the Google python docstring style guide? https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html